provider/scripts/k8s/offline_store_pandas_runner.py

```diff
@@ -318,98 +386,212 @@ def get_code_from_file(mode, file_path, etcd_credentials):
     return code
 
 
-def get_blob_credentials(args):
-    """
-    Retrieve credentials from the blob store.
-    Parameters:
-        args: Namespace (input arguments that passed via environment variables)
-
-    Output:
-        credentials: Namespace(type="", ...) (includes credentials needed for each blob store.)
-    """
-
-    if args.mode == K8S_MODE and args.azure_blob_credentials:
-        return Namespace(
-            type=AZURE,
-            connection_string=args.azure_blob_credentials,
-            container=args.azure_container_name,
-        )
-    else:
-        return Namespace(
-            type=LOCAL,
-        )
-
-
 def get_etcd_host(host, ports):
     """
     Converts the host and ports list into the expected for need for etcd client.
     Parameters:
         host: str   ("127.0.0.1")
         port: [str] (["2379"])
-    Output:
-        etcd_host: [(str, str)] ([("127.0.0.1", "2379")])
+
+    Returns:
+        etcd_host: [(str, str)] (example: [("127.0.0.1", "2379")])

```

Comment:
Do we need any of this etcd stuff in here?

Reply:
Goal was to leverage ETCD for storing transformations but we never implemented the backend (go side) so these are commented out. I can remove it completely as well.